src/components/Settings/Settings.js

```diff
@@ -28,11 +32,16 @@ export default class Settings extends React.Component {
             <option value="nOSLocal">nOS Local</option>
           </select>
         </label>
+        {this.state.saved && <Saved /> }
```

Comment:
Just thinking out loud here.  I think we could do this without the need for setting state.  When the current network changes, the store has progress change from `LOADING` to `LOADED`.  I think we could watch for that change using `withProgress` combined with `componentDidUpdate` checking for a change in progress and updating the view accordingly.

With that said, it's not a fleshed out idea, and I wouldn't let it block merging this.

Reply:
After what I'm doing now I'll have a look into it 😄 Haven't used withProgress yet